wsl/accessibility/ubuntu-wsl-doc-images.md

```diff
@@ -41,123 +41,126 @@ explain what it is and what it does:
 
-![|542x404](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/installing.png?raw=true)
+![NVIDIA Virtual Host controller installation progress](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/installing.png?raw=true)
 
-![|544x406](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/install-finished.png?raw=true)
+![NVIDIA Graphics Driver installation success page](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/install-finished.png?raw=true)
 
-![|544x559](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/done-done.png?raw=true)
+![Terminal output showing successful installation of NVIDIA CUDA toolkit on Ubuntu](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/done-done.png?raw=true)
 
-![|624x135](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/make.png?raw=true)
+![Terminal output showing the successful compilation of a CUDA sample application](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/make.png?raw=true)
 
-![|548x599](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/device-query.png?raw=true)
+![Terminal output showing the results of running the deviceQuery sample application](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/device-query.png?raw=true)
 
 ## Use WSL for data science and engineering
 
 [link to documentation page](https://canonical-ubuntu-wsl.readthedocs-hosted.com/en/latest/tutorials/data-science-and-engineering/)
 
-![|624x373](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/octave.png?raw=true)
+![Windows desktop showing a terminal running "octave --gui" and the Octave GUI](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/octave.png?raw=true)
 
-![|570x225](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/save-file.png?raw=true)
+![Octave GUI showing the "Save and Run button" for the juliatest.m file](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/save-file.png?raw=true)
 
-![|579x540](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/julia-fractal.png?raw=true)
+![A Julia fractal](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/julia-fractal.png?raw=true)
 
 # Howtos
 
 ## Install Ubuntu on WSL2
 
 [Link to documentation page](https://canonical-ubuntu-wsl.readthedocs-hosted.com/en/latest/guides/install-ubuntu-wsl2/)
 
-![|624x489](https://github.com/ubuntu/WSL/blob/main/docs/guides/assets/install-ubuntu-wsl2/choose-distribution.png?raw=true)
+![Installation page for Ubuntu 24.04 LTS on the Microsoft store](https://github.com/ubuntu/WSL/blob/main/docs/guides/assets/install-ubuntu-wsl2/choose-distribution.png?raw=true)
```

Comment:
I prefer "in" to "on" here and below.

Reply:
Any particular reason why? Asking so i'd know how to avoid in the future.